tests/src/test/java/alluxio/shell/AbstractAlluxioShellTest.java

```diff
@@ -93,6 +93,16 @@ protected void copyToLocalWithBytes(int bytes) throws IOException {
     Assert.assertEquals(getCommandOutput(new String[] {"copyToLocal", "/testFile",
         mLocalAlluxioCluster.getAlluxioHome() + "/testFile"}), mOutput.toString());
     fileReadTest("/testFile", 10);
+    mOutput.reset();
```

Comment:
could you move this testing to its own test under `CopyToLocalCommandTest`? This method is supposed to test copying with various numbers of bytes